<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mywindow</class>
 <widget class="QMainWindow" name="mywindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>378</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>이메일 중복제거기</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>250</y>
      <width>63</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>중복 제거</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>59</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1. 파일을 불러오세요.(csv, txt, xlsx) </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>250</y>
      <width>63</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>끝내기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>126</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2. 원하시는 파일 이름을 지정해주세요.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>304</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>카페/블로그/키워드수집기 구매문의</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>87</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 열기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>320</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>wowon7975@naver.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>6</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>이메일 중복 제거</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>42</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio1">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>174</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>csv파일</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio2">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>174</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>txt파일</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>190</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3. 원하시는 저장 방식을 선택해주세요.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>42</y>
      <width>256</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>230</y>
      <width>257</width>
      <height>71</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>250</y>
      <width>63</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>pushButton1</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton0</zorder>
   <zorder>label_12</zorder>
   <zorder>label</zorder>
   <zorder>label_6</zorder>
   <zorder>listWidget</zorder>
   <zorder>listWidget2</zorder>
   <zorder>pushButton2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
